routes/channels.js

```diff
@@ -1,13 +1,36 @@
 const express = require('express');
-const { getChannels } = require('../repositories/channel-repository');
+const { getChannels, getChannelById } = require('../repositories/channel-repository');
+const cors = require('cors');
```

Comment:
@tylerlan I think we're using this high enough in the middleware stack that we won't need it here.